src/mmw/js/src/modeling/views.js

```diff
@@ -1032,24 +1034,54 @@ var ResultsDetailsView = Marionette.LayoutView.extend({
 
     initialize: function(options) {
         this.scenario = options.scenario;
+        this.showHydrologyAndWaterQualityResults = this.showHydrologyAndWaterQualityResults.bind(this);
+        this.showSubbasinHotSpotView = this.showSubbasinHotSpotView.bind(this);
+        this.hideSubbasinHotSpotView = this.hideSubbasinHotSpotView.bind(this);
     },
 
     regions: {
+        subbasinRegion: '.subbasin-region',
         panelsRegion: '.tab-panels-region',
         contentRegion: '.tab-contents-region'
     },
 
     onShow: function() {
+        this.showHydrologyAndWaterQualityResults();
+    },
+
+    showHydrologyAndWaterQualityResults: function() {
         this.panelsRegion.show(new ResultsTabPanelsView({
-            collection: this.collection
+            collection: this.collection.withoutSubbasin(),
         }));
 
         this.contentRegion.show(new ResultsTabContentsView({
             collection: this.collection,
             scenario: this.scenario,
-            areaOfInterest: this.options.areaOfInterest
+            areaOfInterest: this.options.areaOfInterest,
+            showSubbasinHotSpotView: this.showSubbasinHotSpotView,
         }));
-    }
+
+        return this;
+    },
+
+    showSubbasinHotSpotView: function() {
+        this.panelsRegion.empty();
+        this.contentRegion.empty();
+
+        this.subbasinRegion.show(new gwlfeSubbasinViews.ResultView({
+            hideSubbasinHotSpotView: this.hideSubbasinHotSpotView,
+            model: this.collection.findWhere({ name: 'subbasin' }),
```

Comment:
Good catch -- I removed the mock subbasin model (the stray "remove this" comment was supposed to go with it) since we currently don't persist the results.